CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateRecorder.swift

```diff
@@ -486,20 +474,21 @@ public class CRFHeartRateRecorder : RSDSampleRecorder, CRFHeartRateVideoProcesso
                 // get the red channel and the uptime then remove the first half the samples
                 let halfLength = windowLength / 2
                 let uptime = self.pixelSamples[Int(halfLength)].uptime
-                let channel = self.pixelSamples[..<windowLength].map { $0.green }
+                let channel = self.pixelSamples[..<windowLength].map { $0.red }
                 self.pixelSamples.removeSubrange(..<halfLength)
                 
                 self.sampleProcessingQueue.async {
                     
                     let (heartRate, confidence) = calculateHeartRate(channel)
+                    
                     let bpmSample = CRFHeartRateBPMSample(uptime: uptime, bpm: heartRate, confidence: confidence)
                     self.bpmSamples.append(bpmSample)
                     self.isProcessing = false
                     
                     if confidence > CRFMinConfidence {
                         DispatchQueue.main.async {
                             self.confidence = confidence
-                            self.bpm = heartRate
+                            self.bpm = Int(heartRate)
```

Comment:
at first I was thinking this should round rather than truncate, but then I saw below that the calculateHeartrate() func already does that--wondering if there should be a comment here mentioning that, or if the doc comments on calculateHeartrate() should mention that although it returns a double, it's been rounded?